app/components/point-group.jsx

```diff
@@ -145,14 +146,11 @@ const PointGroup = props => {
                     },
                     [undefined, []]
                   )
-                  const newPointObj = {
-                    ...p,
-                    groupedPoints: g,
-                  }
-                  setPointObj(newPointObj)
+                  const newPoint = <Point point={p} groupedPoints={g} />
```

Comment:
on 149, the new code is setting newPoint to a react component `<Point...>`  but this needs to be a point object, that is the point property passed into a `<Point>`

I had trouble with the "point" terminology and that's why I started referring to pointObj to help keep is strait. 

Other developers have had this sort of mix up.  I welcome ways to make it clearer. 

There are other places in this module where this comment applies.

Reply:
Ah okay, that clears things up and seems to have fixed the empty component issue, thanks! Would var. names like _setPointData_ and _newPointData_ help with disambiguation? I also added comments next to wherever that function's called.

Reply:
how about pointDoc - what javascipt calls an object, MongoDb calls a document.  And doc is shorter.